app/services/medical_copays/vbs/service.rb

```diff
@@ -57,6 +62,25 @@ def get_copays
         ResponseData.build(response:).handle
       end
 
+      def get_cached_copay_response
+        StatsD.increment("#{STATSD_KEY_PREFIX}.init_cached_copays.fired")
+
+        cached_response = get_user_cached_response
+        if cached_response
+          StatsD.increment("#{STATSD_KEY_PREFIX}.init_cached_copays.cached_response_returned")
+          return cached_response
+        end
+
+        response = get_copay_response.body
+
+        if response.is_a?(Array) && response.empty?
+          StatsD.increment("#{STATSD_KEY_PREFIX}.init_cached_copays.empty_response_cached")
+          Rails.cache.write("vbs_copays_data_#{user.uuid}", response, expires_in: time_until_5am_utc)
```

Comment:
I wonder, did we ever find out how often the VBS data is refreshed? I forget 🤔 .

Since this is FF'd I don't want to hold up the PR but we should ask about that.

Reply:
I agree about not letting this hold up the pull request. If it's a different time, we can add a method similar to the current one to the Redis Caching module.